localstack/services/route53resolver/models.py

```diff
@@ -13,3 +14,73 @@ def __init__(self):
         self.firewall_domain_lists = {}
         self.firewall_domains = {}
         self.firewall_rules = {}
+        self.firewall_rule_group_associations = {}
+
+    def get_firewall_rule_group(self, id):
+        firewall_rule_group = self.firewall_rule_groups.get(id)
+        if not firewall_rule_group:
+            raise ResourceNotFoundException(
+                f"Can't find the resource with ID '{id}'. Trace Id: '{aws_stack.get_trace_id()}'"
+            )
+        return firewall_rule_group
+
+    def delete_firewall_rule_group(self, id):
```

Comment:
nit: I'd add some type hints for these utility functions

Reply:
Added comments